### PR TITLE
improve Blitz Javadoc

### PR DESCRIPTION
**Pull request by mtbc**
This PR should improve Blitz's Javadoc which should now report warnings from only,

* `ome.services.*`
* `omero.gateway.*`
* `pojos.*`

--no-rebase

### CODE REVIEW

**Comment by jburel** [comment]
warning from 
 * omero.gateway.*
 * pojos.*
taken care of in https://github.com/openmicroscopy/openmicroscopy/pull/4128

**Comment by mtbc** [comment]
:+1:

File: components/blitz/src/omero/client.java [hunk 1]
```diff
@@ -419,9 +428,11 @@ private void init(Ice.InitializationData id) {
     }
 
     /**
-     * Sets the {@link #fastShutdown} flag. By setting this
+     * Sets the {@code fastShutdown} flag. By setting this
```
Comment (joshmoore) [inline components/blitz/src/omero/client.java]: Is this again because you have something set to `protected`, etc?

Comment (mtbc) [inline components/blitz/src/omero/client.java]: `fastShutdown` is `private`.

**Comment by joshmoore** [comment]
All looks good. I'm certainly guilty of having leaked implementation details in javadoc. Down the road, rather than just `s/link/code`'ing, we might want to have a more comprehensive strategy.

**Comment by mtbc** [comment]
Thank you. Yes, some of the Javadoc is quite keen on the "how" as well as the "what". :smiley:

**Pull request merged**